core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -148,6 +160,13 @@ oppia.factory('oppiaPlayerService', [
       LearnerParamsService.init(startingParams);
     };
 
+    var _checkTimeSinceStart = function(totalTimeSecs, stateName) {
```

Comment:
This "check" is doing stuff. That's not a check.

Why don't you use a $timeout that gets set up when the exploration first loads? After 60 seconds it fires an event, and then that's it. (Just to check, though, is your condition for "actually start" that the learner is on the page for 60 seconds, or do you also stipulate that they must advance by at least one state? This needs to be spelled out in docstrings/comments. From reading the code, I can't really tell.)

Reply:
@seanlip Can I have a $timeout block which would execute independent of the other stuff (again, not much experience in Angular). Oh, and 60 seconds alone is not the condition, so it shouldn't be triggered after 60 seconds. I did those checks after state transitions so that the learner progresses states as well as stays on the exploration for 60 seconds.

Reply:
OK, I think I'm a little confused about the requirements here then. Can you state precisely what the condition/action is that should trigger a "really started" call?

Reply:
Redundant since we decided exploration actual start is the state transition from initial state.